lib/child_process.js

```diff
@@ -136,7 +136,7 @@ function normalizeExecArgs(command, options, callback) {
   }
 
   // Make a shallow copy so we don't clobber the user's options object.
-  options = Object.assign({}, options);
```

Comment:
i realize this isn't directly related but it would be good to avoid assigning to the argument.

Reply:
We do that pretty often everywhere throughout the code?

Reply:
could be a separate pr i guess... i just figured as long as you're modifying a bunch of these you could change the argument name. non blocking.